packages/lit-dev-content/site/blog/2022-07-19-lit-for-polymer-users.md

```diff
@@ -0,0 +1,935 @@
+---
+tags: blogPosts
+layout: article.html
+title: "Lit for Polymer users"
+summary: "Migrating Polymer apps to Lit, or learning Lit as a Polymer user"
+date: 2022-07-19
+author:
+  - arthur-evans
+---
+
+Lit is a successor to the Polymer library. If you have a project built with Polymer and want to migrate it to Lit, or if you're familiar with Polymer and want to know how it compares to Lit, this is the right document for you.
```

Comment:
Since we're no longer using the "Polymer project" brand, and have disassociated Lit from it, would it make sense to start referring to Polymer simply as "Polymer" again, instead of "the Polymer library"? It doesn't really feel like the "Polymer library" name caught on in our community.

Reply:
I'm not sure about dropping it altogether; but for the sake of brevity I use "Polymer" in most of this article. I updated the text to be more consistent, and only use the full name on the first reference.